src/utils/setup/create-podcast-pages.ts

```diff
@@ -1,23 +1,31 @@
 import path from 'path';
 import { getTagPageUriComponent } from '../get-tag-page-uri-component';
-import { Podcast } from '../../interfaces/podcast';
+import { Podcast } from '../../classes/podcast';
+import { getSeriesPodcasts } from '../get-series-podcasts';
 import { formatDateToPublishDateFormat } from '../format-dates';
+import { getRelatedPagesWithImages } from './get-related-pages-with-images';
        
 export const createPodcastPages = async (
     createPage: Function,
     allPodcasts: Podcast[],
+    slugContentMapping: any,
+    podcastSeries: object,
     metadata: object
 ) => {
     allPodcasts.forEach((podcast: Podcast) => {
-        const slug = `/podcasts/${getTagPageUriComponent(podcast.title)}`;
-        podcast.slug = slug;
+        podcast.related = getRelatedPagesWithImages(
+            podcast.related,
+            slugContentMapping
+        );
+        const seriesPodcasts = getSeriesPodcasts(podcastSeries, podcast['slug']);
         podcast.publishDate = formatDateToPublishDateFormat(new Date(podcast.publishDate));
         createPage({
-            path: slug,
+            path: podcast['slug'],
```

Comment:
Maybe I am missing something but how does `/podcasts` get applied in the url? (We had it on line 12 before)